patches/tModLoader/Terraria/release_extras/LaunchUtils/InstallNetFramework.sh

```diff
@@ -30,6 +30,12 @@ if [ -d "$dotnet_dir" ]; then
 			rm -rf "$old_version"
 		fi
 	done
+
+	if [[ ! -f "$LaunchLogs/client.log" && ! -f "$LaunchLogs/server.log" ]]; then
+		echo "Last Run Attempt Failed to Start tModLoader. Deleting dotnet_dir and resetting"
+		rm -rf "$dotnet_dir"
+		mkdir "$dotnet_dir"
+	fi
```

Comment:
This belongs in `ScriptCaller.sh` then there's no need to `export LaunchLogs`. Just do the `rm -rf` and let the `InstallNetFramework` call do the `mkdir`